pages/common/routers/success.js

```diff
@@ -24,22 +24,23 @@ module.exports = ({
   model = 'model',
   licence,
   type,
-  status
+  status,
+  refModel
```

Comment:
What is a `refModel`?

Reply:
the model we care about - the one which has openTasks etc - usually req.model, but not in the case of projects - req.model === version, but we care about the project